easybuild/easyconfigs/t/TensorFlow/TensorFlow-2.5.0_fix-arm-vector-intrinsics.patch

```diff
@@ -0,0 +1,37 @@
+The comment is not true, the function actually takes the arguments as it should
+Hence just redefine the function
```

Comment:
@Flamefire Are you planning to open an upstream fix/PR for this?

Reply:
Not really. I guess that comment is true for their environment. Could you open an upstream issue with that? They need details about the environment and as we don't have an ARM cluster I can't really contributed there.

Might also be worth testing this fix first and mentioning it if it does work. I was just guessing here.